frontend/public/components/resource-dropdown.tsx

```diff
@@ -46,7 +46,7 @@ const DropdownItem: React.SFC<DropdownItemProps> = ({model, showGroup}) => <Reac
     </span>
     <span className="co-resource-link__resource-name">
       {model.kind}
-      {showGroup && <React.Fragment>&nbsp;<small className="text-muted">&ndash; {model.apiGroup || 'core'}/{model.apiVersion}</small></React.Fragment>}
+      {showGroup && <React.Fragment>&nbsp;&ndash;<small className="text-muted truncate">{model.apiGroup || 'core'}/{model.apiVersion}</small></React.Fragment>}
```

Comment:
Rather than adding a new class (`truncate`), I recommend using `co-truncate co-nowrap show` to leverage existing CSS rules and achieve the same result.